web_widget_image_webcam/static/src/xml/web_widget_image_webcam.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Siddharth Bhalgami <siddharth.bhalgami@techreceptives.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<template>
+    <t t-extend="FieldBinaryImage">
+        <t t-jquery=".oe_form_field_image_controls" t-operation="after">
+            <div class="oe_form_field_image_web_cam_controls oe_edit_only">
+              <a class="fa fa-eye fa-1g col-md-4 oe_form_binary_file_web_cam"
+                 data-toggle="modal" data-target="#WebCamModal" title="WebCam"></a>
+            </div>
+            <div id="WebCamModal" class="modal fade">
+                <div class="modal-dialog modal-lg">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <button type="button" class="close" data-dismiss="modal">x</button>
```

Comment:
Use a `fa-times` icon better.

[Remember to call `Webcam.reset()` when this modal is closed](https://github.com/jhuckaby/webcamjs#reset-shutdown).